stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -221,6 +221,24 @@ spec:
             failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
             periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
             successThreshold: {{ .Values.livenessProbe.successThreshold }}
+{{- if semverCompare ">=1.16-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Looks like startupProbe is not enabled by default (e.g. marked beta) until 1.18. Would prefer this be set to that.